app/views/steps/income/client/other_work_benefits/edit.html.erb

```diff
@@ -1,22 +1,23 @@
-<% title t('steps.income.caption') %>
+<% title t('.page_title') %>
 <% step_header %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-xl"><%= t('steps.income.caption') %></span>
     <%= govuk_error_summary(@form_object) %>
-
-    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
-    <p class="govuk-body">
-      <%= t('.page_text') %>
-    </p>
-    <ul class="govuk-body govuk-list--bullet govuk-!-margin-bottom-6">
-      <% t('.list').each_line do |line| %>
-        <li><%= line %></li>
-      <% end %>
-    </ul>
     <%= step_form @form_object do |f| %>
-      <%= f.govuk_radio_buttons_fieldset :applicant_other_work_benefit_received, legend: { tag: 'h1', size: 'xl', hidden: true } do %>
+      <%= f.govuk_radio_buttons_fieldset :applicant_other_work_benefit_received,
+                                         legend: { text: t('.applicant_other_work_benefit_received'),
+                                                   tag: 'h1', size: 'xl' } do %>
+        <p class="govuk-body">
+          <%= t('.page_text') %>
+        </p>
+        <ul class="govuk-body govuk-list--bullet govuk-!-margin-bottom-6">
+          <% t('.list').each_line do |line| %>
+            <li><%= line %></li>
+          <% end %>
+        </ul>
+
         <%= f.govuk_radio_button :applicant_other_work_benefit_received, YesNoAnswer::YES do %>
```

Comment:
This was amended to work with the current spec helpers where the radio button is nested in the same element as the label. It has no front end impact